src/core/services/database/sites.ts

```diff
@@ -15,14 +15,29 @@
 import { CoreAppSchema } from '@services/app';
 import { CoreSiteSchema } from '@services/sites';
 import { SQLiteDB, SQLiteDBTableSchema } from '@classes/sqlitedb';
-import { CoreSite } from '@classes/site';
 
 /**
  * Database variables for CoreSites service.
  */
 export const SITES_TABLE_NAME = 'sites_2';
 export const SCHEMA_VERSIONS_TABLE_NAME = 'schema_versions';
 
+/**
+ * Database variables for CoreSite class.
+ */
+export enum WSCacheTablesSuffixes {
+    CORE = 'core',
+    BLOCK = 'block',
+    MOD = 'mod',
+    TOOL = 'tool',
+    OTHER = 'other',
+}
+export const WS_CACHE_OLD_TABLE = 'wscache_2';
+export const WS_CACHE_TABLES_COMMON = 'wscache_';
```

Comment:
```suggestion
export const WS_CACHE_TABLES_PREFIX = 'wscache_';
```
What do you think of this? It may seem like I am contradicting my previous comment, but in this case I think this variable really *is* the table prefix, I don't think it carries any other meaning.